gbfs-validator/cli.js

```diff
@@ -1,17 +1,95 @@
+#!/usr/bin/env node
+
 const { inspect } = require('util')
-const GBFS = require('./')
+const commander = require('commander')
+const fs = require('fs')
+const fsPath = require('path')
+const GBFS = require('./gbfs')
+const pjson = require('./package.json')
+
+getFeedValidationReport = async (url) => {
+  const gbfs = new GBFS(url)
+  return gbfs.validation()
+}
+
+const isExitOverrided = () => (process.env.EXIT_OVERRIDE === 'true')
 
-if (!process.argv[2]) {
-  console.error('Usage: gbfs-validator GBFS_URL')
-  process.exit(1)
+const printingReport = (options) => (options.printReport === 'yes')
+
+const exitProcess = (code) => {
+  if (!isExitOverrided && code === 1) {
+    process.exit(code)
+  }
+  process.exit(0)
 }
 
-const gbfs = new GBFS(process.argv[2])
+parseOptions = () => {
+  commander
+    .version(pjson.version, '-v, --version')
+    .usage('[OPTIONS]...')
+    .requiredOption('-u, --url [feed_url]', 'URL of the GBFS feed')
```

Comment:
I suggest using angle brackets here since the `feed_url` is required.

According to the [documentation](https://www.npmjs.com/package/commander#other-option-types-negatable-boolean-and-booleanvalue), square brackets are used for optional arguments.

>You can specify an option which may be used as a boolean option but may optionally take an option-argument (declared with square brackets like --optional [value]).

```suggestion
    .requiredOption('-u, --url <feed_url>', 'URL of the GBFS feed')
```

Reply:
Nice catch! 🥳 Done